### PR TITLE
Ns/remove pillow0 stagin

### PR DESCRIPTION
**Pull request by NitigyaS**
##### SUMMARY

Removes older pillow0 from staging. 

##### ISSUE TYPE

- Change Pull Request
- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

* Before
```paste below

```

* After
```paste below

```

### CODE REVIEW

File: environments/india/inventory.ini [hunk 1]
```diff
@@ -12,6 +12,39 @@ datavol_device='/dev/mapper/consolidated-data'
...
+[citusdb_worker:vars]
+ansible_python_interpreter = '/usr/bin/python3'
+datavol_device=/dev/xvdc
+
+[citusdb_cordinator:children]
+citusdb0
+
+[citusdb_cordinator:vars]
+ansible_python_interpreter = '/usr/bin/python3'
+datavol_device=/dev/xvdc
```
Comment (snopoke) [inline environments/india/inventory.ini]: these changes are unrelated to removing the pillow

Comment (NitigyaS) [reply environments/india/inventory.ini]: checkout from wrong branch.

File: environments/staging/inventory.ini [hunk 2]
```diff
@@ -88,7 +88,6 @@ celery1
 
 [pillowtop:children]
 # Amazon EC2
-pillow0
```
Comment (snopoke) [inline environments/staging/inventory.ini]: I think you need to make this change to `inventory.ini.j2` and also remove the machine from terraform.

Comment (NitigyaS) [reply environments/staging/inventory.ini]: I am not yet delete the VM only removing it from inventory
